docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,149 @@
+| --log-level        | The log level to set for the flow run. If passed, the level must be a valid Python logging level name. If this option is not passed, the default level for the flow will be used. Valid values include DEBUG, INFO, WARNING, ERROR, or CRITICAL. |
+| --param-file       | The path to a JSON file containing parameter keys and values. Any parameters passed with `--param` will take precedence over these values. |
+| <span class="no-wrap" style="white-space:nowrap;">--idempotency-key</span>  | A key to prevent duplicate flow runs. If a flow run has already been started with the provided value, the command will display information for the existing run. If using `--execute`, duplicate flow runs will exit with an error. If not using the backing API, this flag has no effect. |
+| --execute          | Execute the flow run in-process without an agent. If this process exits, the flow run will be marked as 'Failed'. |
+| -s, --schedule     | Execute the flow run according to the schedule attached to the flow. If this flag is set, this command will wait between scheduled flow runs. If the flow has no schedule, this flag will be ignored. If used with a non-local run, an exception will be thrown. |
+| -q, --quiet        | Disable verbose messaging about the flow run and just print the flow run ID. |
+| --no-logs          | Disable streaming logs from the flow run to this terminal. Only state changes will be displayed. Only applicable when `--watch` is set. |
+| -w, --watch        | Wait for the flow run to finish executing and display status information. |
+
+For example, run a flow in a script locally:
+
+```bash
+$ prefect run -p hello-world.py
+```
+
+Run a flow with a non-default parameter locally:
+
+```bash
+$ prefect run -m prefect.hello_world --param name=Marvin
+```
+
+Run a registered flow with the backend with custom labels:
+
+```bash
+$ prefect run -n "hello-world" --label example --label hello
+```
+
+Run a registered flow and execute locally without an agent
+
+```bash
+$ prefect run -n "hello-world" --execute
+```
+
+## Imports have moved
+
+Imports for some Prefect modules have moved:
+
+- Artifacts functions now imported from `prefect.backend.artifacts`.
+- `Parameter` now imported from `prefect.Parameter` instead of `prefect.core.tasks`.
+- Exceptions now imported from `prefect.exceptions` instead of `prefect.utilities.exceptions`.
+- Executors now imported from `prefect.executors` instead of `prefect.engine.executors`. 
+
+## iCal recurrence rules schedules
+
+Prefect now supports rich recurrence rule scheduling following the iCal RRules standard and `dateutil` `rrule` module. This feature does not impact existing schedules using interval clocks, cron clocks, and so on, but provides convenient, new syntax for creating repetitive schedules. See [Recurrence Rule Clocks](/core/concepts/schedules.html#recurrence-rule-clocks) for details.
```

Comment:
I would like to add community contributed and link the PR. The PR is [here](https://github.com/PrefectHQ/prefect/commit/7293bfd667143e40c9d648d3c12845e3298e49f1#diff-72230825c86952868b5f8b493a265e20b5091f832144fa52d1592fb8c05dd167) and is one of the biggest community contributions I have personally seen.

Reply:
too much? 

Thanks to Prefect community member [Nitay Joffe](https://github.com/nitay) for this significant contribution to add [Support for RRule (iCal style recurrence rule) clocks/schedules](https://github.com/PrefectHQ/prefect/pull/4901).

Reply:
We have the community-contributed in Orion docs and we don't mention him by name haha. I don't have an opinion either way though.